styles/sass/buttons/toggle-buttons.scss

```diff
@@ -0,0 +1,201 @@
+$toggleButtonTextColor: #5a5a5a;
+$toggleButtonBorderColor: #c6c6c6;
+
+.toggle-button {
+    display: inline-block;
+    text-decoration: none;
+    border-radius: 2px;
+    box-sizing: border-box;
+    padding: 2px 10px;
+    text-decoration: none;
+    font-size: 12px;
+    background: transparent;
+    color: $toggleButtonTextColor;
+    border: 1px solid $toggleButtonBorderColor;
+    height: 26px;
+    line-height: 20px;
+    margin-bottom: 4px;
+    @include transition(all 0.2s ease-out);
+    &:hover {
+        text-decoration: none;
+        color: darken($toggleButtonTextColor, 30%);
+        border-color: darken($toggleButtonBorderColor, 30%);
+        @include box-shadow(0 1px 4px rgba(0,0,0,0.20));
+    }
+    &:active {
+        @include box-shadow(inset 0 2px 0 rgba(0,0,0,0.20));
+        border-color: darken($toggleButtonBorderColor, 30%);
+    }
+    .toggle-button__icon {
+        color: $toggleButtonTextColor;
+        line-height: 20px;
+    }
+}
+
+.toggle-button--active {
+    color: $white;
+    background-color: darken($sd-blue, 11%);
+    border-color: darken($sd-blue, 11%);
+    border-top-color: darken($sd-blue, 30%);
+    @include box-shadow(inset 0 2px 0 rgba(0,0,0,0.20));
+    &:hover {
+        color: $white;
+        border-color: darken($sd-blue, 16%);
+        background-color: darken($sd-blue, 16%);
+        @include box-shadow(inset 0 2px 0 rgba(0,0,0,0.20));
+    }
+    &:active {
+        @include box-shadow(inset 0 2px 0 rgba(0,0,0,0.20));
+        border-color: darken($sd-blue, 11%);
+        background-color: darken($sd-blue, 1%);
+    }
+    .toggle-button__icon {
+        color: $white;
+    }
+}
+
+.toggle-button__text {
+    line-height: 22px;
+}
+
+.button-list {
+    overflow: hidden;
+   .toggle-button, .btn {
+       float:left;
+       margin-right: 6px;
+   } 
+}
+.button-list--padded {
+   padding: 11px 0 11px 20px;
+}
+
+// .toggle-list {
```

Comment:
I think it would be a good idea to make an effort and not commit any commented out code (or styling) into the `master` branch...